### ISSUE TITLE
Ideas Needed: Specific Job Designations

### ISSUE THREAD

**Issue by Pyko1** [HippieStation/HippieStation13#493]
The Idea is to have a pool of job names from where it randomizes the job name to PDA/ID, and HoP can choose the designation when changing job.
ie. 
Janitor: Assistant/Junior/Senior Toilet Sanitation Technician, Broom Technician, Mop Manager
Engineer: Tesla Tech, Singularity Mechanic, Wrench Monkey
Atmos Tech: Plumber, Air Duct Augmenter
Chef: Chef, Cook, Burger Flipper
Medical Doctor: Flatologist, Proctologist, Urologist, Gynecologist, Gastroenterologist
Cargo Technician: Senior Stamp Stamper, Postal Boy, Logistic Manager

**Comment by Pyko1** [comment]
Definitely needs weight to make shit more rare, and possibly even restrict the job names too rare from hop.

**Comment by Pyko1** [comment]
Assistant: Tide Technician

**Comment by JohnGinnane** [comment]
Good idea, I just worry that the names might be confusing for new players

**Comment by Sarcalogo** [comment]
They should actually represent what it is players are doing instead of being stupid memes. I don't want to be represented as "Urologist" when Urology doesn't even exist in the game. Here are some examples.

Medical Doctor: Paramedic, Surgeon, General Practitioner
Scientist: Xenobiologist, Researcher, Toxins
Engineer: Electrician, Maintenance Worker
Security Officer: Officer, Guard

**Comment by JohnGinnane** [comment]
Didn't para use something like this? You could choose the job name when selecting your job in character setup
